### ISSUE TITLE
Themes: "theme image more" is read aloud for odd-numbered items in the list

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-Android#10939]
Steps to reproduce:

1. Go to Settings > Accessibility > TalkBack > "Use service" or use the [Volume key shortcut](https://support.google.com/accessibility/android/answer/6006966?hl=en).
1. In the WordPress app, go to My Sites > Themes.
1. Swipe right or tap on different themes.

**Result:** "theme image more" is read aloud for odd-numbered items in the list. ([1m17s](https://cloudup.com/c_c9Bpikx9v))

<img src="https://user-images.githubusercontent.com/1119271/70569012-54ad4380-1b56-11ea-873e-a08366866a15.png" width="270" alt="themes-screen">
<sup>Tested with WPAndroid 13.8-rc-1 on Pixel 3 Android 10.</sup>

<p style="clear:both;"><sub>/hat tip @mchowning for the report, internal reference: p77Llu-c0a-p2</sub></p>

**Comment by shiki** [comment]
@jd-alexander Are you still working (or planning) to work on this?

**Comment by jd-alexander** [comment]
@shiki Yes, I was gonna find some time to get it done after I was finished with everything else.

**Comment by A-L-Fearon** [comment]
Hi, I would like to take on this issue.

**Comment by jd-alexander** [comment]
Sure @A-L-Fearon go ahead 🙌🏾

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
